pkg/networkmanager/utils.js

```diff
@@ -56,6 +56,35 @@ export function ip_metric_from_text(text) {
     throw cockpit.format(_("Invalid metric $0"), text);
 }
 
+export function ip_network_address(address, prefix_len) {
+    const addrCIDR = address.toString() + "/" + prefix_len;
+    const ip_obj = address.kind() === "ipv4" ? ipaddr.IPv4 : ipaddr.IPv6;
+
+    try {
+        return ip_obj.networkAddressFromCIDR(addrCIDR).toString();
+    } catch (_e) {
+        return null;
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22608-f285fbdb-20251124-105506-fedora-43-devel/Coverage/lcov/github-pr.diff.gcov.html)